src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const largeImage = document.getElementById('largeImg');
+const listGallery = document.querySelector('.gallery__list');
+
+listGallery.addEventListener('click', (currentElement) => {
```

Comment:
Consider renaming `currentElement` to `event` to better reflect its role as the event object. This will improve code readability and understanding.